at_lookup/lib/src/connection/outbound_sync_message_listener.dart

```diff
@@ -0,0 +1,54 @@
+import 'dart:collection';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:at_lookup/src/connection/outbound_message_listener.dart';
+
+class SyncMessageListener extends OutboundMessageListener {
+  Queue pendingData = Queue<List<int>>();
+  var atConst = '@'.codeUnitAt(0);
+  var dollarConst = '\$'.codeUnitAt(0);
+
+  SyncMessageListener(connection) : super(connection);
+
+  @override
+  Future<void> messageHandler(sync_data) async {
+    // If sync_data contains only '@', return.
+    if (sync_data.length == 1 && sync_data.first == atConst) {
+      return;
+    }
+    // if sync_data last is '@' and contains a new line.
+    if (sync_data.last == atConst && sync_data.contains(10)) {
```

Comment:
replace 10 with a constant

Reply:
Replaced with a final variable. Getting compile issue when having it as constant.